code/__HELPERS/traits.dm

```diff
@@ -62,7 +62,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 //mob traits
 #define TRAIT_PACIFISM			"pacifism"
-#define TRAIT_WATERBREATH       "waterbreathing"
+#define TRAIT_WATERBREATH		"waterbreathing"
```

Comment:
uh!

Reply:
Uh yes I can explain, the area between `TRAIT_WATERBREATH` and `"waterbreathing"` were all spaces and it looked ugly, so I converted it to using tabs :smile: